src/app/devices/deviceContent/components/deviceSettings/deviceSetttingsPerInterfacePerSetting.spec.tsx

```diff
@@ -54,7 +54,7 @@ describe('components/devices/deviceSettingsPerInterfacePerSetting', () => {
         settingModelDefinition: propertyModelDefinition,
         settingSchema: propertySchema};
 
-    it('matches snapshot when there is a writable property of simple type without sync status', () => {
+    it('renders when there is a writable property of simple type without sync status', () => {
```

Comment:
Will need more tests to cover displayName and description returning '--' and unit returning something other than '--'.